xbmc/cores/VideoPlayer/DVDClock.cpp

```diff
@@ -39,6 +39,8 @@ CDVDClock::CDVDClock()
   m_systemAdjust = 0;
   m_speedAdjust = 0;
   m_startClock = 0;
+  m_vSyncAdjust = 0;
+  m_frameTime = 1/60 * DVD_TIME_BASE;
```

Comment:
oh, thanks. yes it should know about frametime because DISCONT corrections by audio are only allowed in intervals of frametime